tools/config/go-templates/model_simple.mustache

```diff
@@ -28,15 +28,8 @@ type {{classname}} struct {
 {{/deprecated}}
 	{{name}} {{^required}}{{^isFreeFormObject}}*{{/isFreeFormObject}}{{/required}}{{{dataType}}} `json:"{{baseName}}{{^required}},omitempty{{/required}}"{{#withXml}} xml:"{{baseName}}{{#isXmlAttribute}},attr{{/isXmlAttribute}}"{{/withXml}}{{#vendorExtensions.x-go-custom-tag}} {{{.}}}{{/vendorExtensions.x-go-custom-tag}}`
 {{/vars}}
-{{#isAdditionalPropertiesTrue}}
```

Comment:
these isAdditionalPropertiesTrue blocks seem not to have any effect on generated files